app/src/main/res/layout/fragment_upcoming.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I think that will be better if this is a ConstraintLayout

Reply:
Done